apollo-livedata-support/src/main/java/com/apollographql/apollo/livedata/ApolloExtension.kt

```diff
@@ -0,0 +1,50 @@
+@file:Suppress("unused")
+
+package com.apollographql.apollo.livedata
+
+import android.arch.lifecycle.LiveData
+import com.apollographql.apollo.ApolloCall
+import com.apollographql.apollo.ApolloPrefetch
+import com.apollographql.apollo.ApolloQueryWatcher
+import com.apollographql.apollo.ApolloSubscriptionCall
+import com.apollographql.apollo.cache.normalized.ApolloStoreOperation
+
+/**
+ * The ApolloExtension class provides extensions for converting ApolloCall, ApolloPrefetch and ApolloWatcher types to LiveData
+ * sources.
+ */
+
+/**
+ * Converts an [ApolloCall] to a LiveData.
+ */
+fun <T> ApolloCall<T>.asLiveData(): LiveData<ApolloLiveDataResponse<T>> {
```

Comment:
One more note, to be aligned with Coroutine ext module lets use `to` prefix (`toLiveData`) instead of `as`.

Reply:
:white_check_mark: Thank you! I will be fixed.